apps/nrm/fields/index.js

```diff
@@ -695,11 +695,53 @@ module.exports = {
       className: 'visuallyhidden'
     },
     options: [
-      'yes',
+      {
+        value: 'yes',
+        toggle: 'other-potential-victims-yes-details',
+        child: 'textarea'
+      },
       'no',
-      'not-sure'
+      {
+        value: 'not-sure',
+        toggle: 'other-potential-victims-not-sure-details',
+        child: 'textarea'
+      }
     ]
   },
+  'other-potential-victims-yes-details': {
+    mixin: 'textarea',
+    labelClassName: ['govuk-body govuk-!-font-weight-bold'],
+    'ignore-defaults': true,
+    formatter: ['trim', 'hyphens'],
+    validate: [{ type: 'maxlength', arguments: [15000] }],
+    attributes: [
+      {
+        attribute: 'rows',
+        value: 7
+      }
+    ],
+    dependent: {
+      value: 'yes',
+      field: 'any-other-pvs'
+    }
+  },
+  'other-potential-victims-not-sure-details': {
+    mixin: 'textarea',
+    labelClassName: ['govuk-body govuk-!-font-weight-bold'],
+    'ignore-defaults': true,
+    formatter: ['trim', 'hyphens'],
+    validate: [{ type: 'maxlength', arguments: [15000] }],
+    attributes: [
+      {
+        attribute: 'rows',
+        value: 7
+      }
+    ],
+    dependent: {
+      value: 'not-sure',
+      field: 'any-other-pvs'
+    }
+  },
```

Comment:
This might not work but you get the gist of the idea.  
```suggestion
  const createFieldConfig = (value, field) => ({
  mixin: 'textarea',
  labelClassName: ['govuk-body govuk-!-font-weight-bold'],
  'ignore-defaults': true,
  formatter: ['trim', 'hyphens'],
  validate: [{ type: 'maxlength', arguments: [15000] }],
  attributes: [{ attribute: 'rows', value: 7 }],
  dependent: { value, field }
});

'other-potential-victims-yes-details': createFieldConfig('yes', 'any-other-pvs'),
'other-potential-victims-not-sure-details': createFieldConfig('not-sure', 'any-other-pvs')
```

Reply:
Thanks Sulthan, I tried this out and it does work, so I have amended as suggested